src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -331,15 +332,25 @@ const formConfig = {
             disabilities: {
               items: {
                 'ui:title': disabilityNameTitle,
-                'ui:description': evidenceTypesDescription,
-                'view:vaMedicalRecords': {
-                  'ui:title': 'VA medical records'
-                },
-                'view:privateMedicalRecords': {
-                  'ui:title': 'Private medical records'
-                },
-                'view:otherEvidence': {
-                  'ui:title': 'Lay statements or other evidence'
+                'view:selectableEvidenceTypes': {
+                  'ui:options': {
+                    // Only way to get access to the disability info like 'name' within this nested schema
+                    updateSchema: (form, schema, uiSchema, index) => ({ title: getEvidenceTypesDescription(form, index) }),
```

Comment:
Why do we need to know the disability name? The second parameter of 'ui:validations' is the field data, is that sufficient?

Reply:
This is to show the description for these check boxes. `What supporting evidence do you have that shows how your {disabilityName} has worsened since VA rated your disability?` `ui:validations` isn't related to this particular change.

We need this `updateSchema` now because these `view:` check boxes are now nested inside another object (`view:selectableEvidenceTypes`), and if we kept the `ui:description` the same, it made for some really wonky formatting. Mark and I scratched our head over this one for a while before asking Jeff (who came up with this particular solution).

It's a weird one for sure!